### ISSUE TITLE
max-visible is not a style parameter and cannot be redefined in criteria section

### ISSUE THREAD

**Issue by TLogiviere** [emersion/mako#124]
Hello again :smile: 
In the continuity of #123, what do you think about allowing `max-visible` parameter to be redefined in criteria section ?
It would permit to prevent some apps congesting the display area with irrelevant past activity, like when one has notifications of sound level changes enabled.

**Comment by vilhalmer** [comment]
I have considered this in the past, and determined that it would make things more complex than I'd like when determining which notifications should be visible. However, now that grouping exists, you can simulate it. For example, this would make two notifications from Firefox visible at a time (index 0 is visible by default):
```
[app-name=Firefox]
group-by=app-name

[app-name=Firefox group-index=1]
invisible=false
```
This does get a little weird when interacting with max-visible at the moment, I'm hoping to improve it in the future.

In the case of things like sound level, the right thing to do is to use a notify client which is able to use replaces_id, like [this one](https://github.com/vlevit/notify-send.sh). Then each sound change just updates the same notification.

**Comment by TLogiviere** [comment]
Just integrated notify-send.sh to my setup and it works perfectly.
Thank you for the pointer !

**Comment by emersion** [comment]
Implemented in #228
